### PR TITLE
Add OpenExpo video

### PR DESCRIPTION
**Pull request by SashoStoichkov**

### CODE REVIEW

File: theme/static/style/style.css [hunk 1]
```diff
@@ -430,6 +430,21 @@ pre code {
...
+    background-repeat: no-repeat;
+    display: inline-block;
+    float: left;
+    margin-left: 1.5rem;
+}
+
+.openexpo-img a {
+    padding-top: 10px;
+    float: right;
+}
```
Comment (atodorov) [inline theme/static/style/style.css:430]: This is exactly the same as `.fosdem-img a`, use a generic class name for the two and don't repeat.

File: theme/static/style/style.css [hunk 2]
```diff
@@ -430,6 +430,21 @@ pre code {
...
 
+.openexpo-img {
+    width: 200px;
+    height: 200px;
+    background-image: url('/images/conf/openexpo.png');
+    background-repeat: no-repeat;
+    display: inline-block;
+    float: left;
+    margin-left: 1.5rem;
+}
```
Comment (atodorov) [inline theme/static/style/style.css]: With the exception of `background-image` this is the same as `fosdem-img`. Use a generic class definition for the layout + a specific one for the background image and don't repeat.

**Review by atodorov — CHANGES_REQUESTED** [review]

**Comment by SashoStoichkov** [comment]
Updated css classes

**Comment by SashoStoichkov** [comment]
I deleted all the other branches. I'll change the code in video branch

**Review by atodorov — APPROVED** [review]

**Pull request merged**